docs/source/production.rst

```diff
@@ -106,6 +106,18 @@ SQLFluff comes with two `pre-commit`_ hooks:
 * sqlfluff-lint: returns linting errors.
 * sqlfluff-fix: attempts to fix rule violations.
 
+.. note::
+   For safety reasons, ``sqlfluff-fix`` by default will not make any fixes in
+   files that had parse errors.
```

Comment:
Is it same for templating errors? Or just parse errors?

Reply:
Thus far, I've only touched the behavior for parse errors. Let me think about and experiment with templating errors. They should probably behave the same way.

Reply:
Ok, I updated it to handle templater errors the same way. Need to review a little more closely to see if there's leftover code to clean up, but it should be working correctly and consistently now.